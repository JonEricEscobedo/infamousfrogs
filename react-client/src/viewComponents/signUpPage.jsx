import React from 'react';


const signUpPage = (props) => (
  <div className="container">
      <div className="row">
      <div className="span12">
        <form className="form-horizontal" action='/register' method="POST">
          <fieldset>
            <div id="legend">
              <legend className="">Sign Up</legend>
            </div>
            <div className="control-group">
              <label className="control-label" htmlFor="username">Username</label>
              <div className="controls">
                <input type="text" id="username" name="username" placeholder="" className="input-xlarge" />
              </div>
            </div>
            <div className="control-group">
              <label className="control-label" htmlFor="password">Password</label>
              <div className="controls">
                <input type="password" id="password" name="password" placeholder="" className="input-xlarge" />
              </div>
            </div>
            <div className="control-group">
              <div className="controls">
                <button className="btn btn-success">Sign Up</button>
              </div>
            </div>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
);

export default signUpPage;
