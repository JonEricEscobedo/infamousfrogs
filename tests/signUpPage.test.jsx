import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json'
import signUpPage from '../react-client/src/viewComponents/signUpPage.jsx';

test('signUpPage component should render as expected', () => {
  const component = shallow(<signUpPage />)
  const tree = toJson(component)

  expect(tree).toMatchSnapshot()
})