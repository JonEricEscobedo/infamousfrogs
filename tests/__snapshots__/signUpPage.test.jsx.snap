// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`signUpPage component should render as expected 1`] = `<signUpPage />`;
